### PR TITLE
housekeeping: cleanup logs, tsc errors, global requires

### PR DESCRIPTION
**Pull request by guymguym**
### Explain the changes
1. cleanup logs, tsc errors, global requires

### Issues: Fixed #xxx / Gap #xxx
1. NA

### Testing Instructions:
1. NA

- [ ] Doc added/updated
- [ ] Tests added

### CODE REVIEW

File: src/sdk/namespace_s3.js [hunk 1]
```diff
@@ -692,7 +700,13 @@ class NamespaceS3 {
 
     /**
      * 
-     * @param {AWS.S3.HeadObjectOutput | AWS.S3.GetObjectOutput} res 
+     * @param {Omit<Partial<
```
Comment (shirady) [inline src/sdk/namespace_s3.js:703]: `Omit` is from typescript? (it is a javascript file I don't understand).
And another question - did you use an automatic tool for all the changes that are JSDoc comments?

File: src/sdk/object_sdk.js [hunk 2]
```diff
@@ -307,6 +304,7 @@ class ObjectSDK {
 
     _setup_merge_namespace(bucket) {
         let rr = _.cloneDeep(bucket.namespace.read_resources);
+        /** @type {nb.Namespace} */
```
Comment (shirady) [inline src/sdk/object_sdk.js:307]: This `@type` is for the `wr` variable?
If so, why not add the `@type` line before `rr` variable statement?

**Review by liranmauda — APPROVED** [review]
LGTM

File: src/sdk/object_sdk.js (hunk 2 above)
Comment (guymguym) [reply src/sdk/object_sdk.js:307]: leaving something for others :)

File: src/sdk/namespace_s3.js (hunk 1 above)
Comment (guymguym) [reply src/sdk/namespace_s3.js:703]: This is a jsdoc with typescript.
https://www.typescriptlang.org/docs/handbook/type-checking-javascript-files.html#supported-jsdoc

Comment (guymguym) [reply src/sdk/namespace_s3.js:703]: the tool is tsc

**Pull request merged**